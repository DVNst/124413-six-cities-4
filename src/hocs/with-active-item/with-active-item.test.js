import React from 'react';
import renderer from 'react-test-renderer';

import withActiveItem from './with-active-item.js';

const MockComponent = () => {
  return (
    <div />
  );
};

const withActiveItemWrapped = withActiveItem(MockComponent);

it(`Render withOpen`, () => {
  const tree = renderer.create(
      <withActiveItemWrapped
        activeItemId={null}
        onOfferCardHover={() => {}}
      />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
