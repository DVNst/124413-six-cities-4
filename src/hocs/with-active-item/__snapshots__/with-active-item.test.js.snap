// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render withOpen 1`] = `
<withActiveItemWrapped
  activeItemId={null}
  onOfferCardHover={[Function]}
/>
`;
